gui/index.go

```diff
@@ -38,6 +38,13 @@ func (ui *GUI) renderIndex(w http.ResponseWriter, r *http.Request, modalError st
 	// time, but the GUI doesn't use them yet.
 	lastPaymentHeight, _, _ := ui.cache.getLastPaymentInfo()
 
+	var address string
+	if ui.cfg.UseLEHTTPS {
```

Comment:
Should use `if ui.cfg.Domain != nil` here.

If somebody is using another webserver like nginx to provide TLS, `ui.cfg.UseLeHTTPS` will be `false` and `127.0.0.1` will be incorrectly shown on the website.

Reply:
Good catch, fixing.

Reply:
I dont get why you've added the extra logic for UseLEHTTPS? Surely just `if ui.cfg.Domain != ""` is enough?

Reply:
It's for the two cases where letsencrypt is being used and when its not, In the case where its not but TLS is still being provided then the domain would be set. In both cases the domain shown should be the domain set.

Reply:
you're right, no need checking the letsencrypt flag to set the domain.